pkg/driver/efs_watch_dog.go

```diff
@@ -78,6 +78,14 @@ tls_cert_renewal_interval_min = 60
 
 [client-info] 
 source={{.EfsClientSource}}
+
+[cloudwatch-log]
+# enabled = true
```

Comment:
wait so is it enabled? Or just placeholder?

Reply:
By default, it is not enabled. However if you set `enabled = true` and have the right cloud watch permissions, efs-utils will start logging to cloud watch.